demo/demo_hqsam.py

```diff
@@ -2,7 +2,7 @@
 import torch
 import matplotlib.pyplot as plt
 import cv2
-from segment_anything import sam_model_registry, SamPredictor
+from segment_anything_hq import sam_model_registry, SamPredictor
```

Comment:
Here we're updating the [Import Package](https://packaging.python.org/en/latest/glossary/#term-Import-Package) in this example to match the name given in the HQ-SAM [documentation](https://github.com/SysCV/sam-hq#quick-installation-via-pip):

https://github.com/SysCV/sam-hq/blob/5fd027ec559806dec9f876b56938175a63dfc625/README.md?plain=1#L90